chall03/llahti.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+func getRgb(str string, start int, end int) string {
+	color, err := strconv.Atoi(str[start:end])
+	if err != nil {
+		log.Fatalln(err)
+	}
+	return fmt.Sprintf("%.2x", color)
+}
+
+func getHTTPResponse(address string) string {
+	resp, err := http.Get(address)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	defer resp.Body.Close()
+
+	response, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	return string(response)
+}
+
+func main() {
+	str := getHTTPResponse("https://chall03.hive.fi/")
```

Comment:
it was a good idea to create your own function, but now if you wanna get some response information like that status code, it's a bit problematic.